packages/blade/src/components/ActionList/StyledActionListItem.web.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+import type { StyledActionListItemProps } from './getBaseActionListItemStyles';
+import { getBaseActionListItemStyles } from './getBaseActionListItemStyles';
+import Box from '~components/Box';
+
+const StyledActionListItem = styled(Box)<StyledActionListItemProps>((props) => {
+  return {
+    ...getBaseActionListItemStyles(props),
```

Comment:
so since we already have a cross platform Box component can't we just have platform dependent styles instead of a styled component all together? so your `StyledActionListItem.web.tsx` can become `getActionListItemStyles.web.ts` and `getActionListItemStyles.native.ts` and you can put the `styled(Box)` directly under `ActionListItem.tsx` file?

Reply:
The element also changes. Here we have `styled(Box)` and on native we have `styled(TouchableOpacity)`